the-depot/Keys.json

```diff
@@ -0,0 +1,14 @@
+﻿[
+  {
+    "Key": "123456789876543234567887654345678",
+    "Used": false,
+    "UsedOnDate": "2020-01-01",
```

Comment:
Don't think this is necessary because we get a new keys file every day from the department head